config_example.py

```diff
@@ -18,18 +24,32 @@
               "For now it downloads music from YouTube or SoundCloud "
               "whenever you send the link and makes new members go through "
               "captcha." + "\n\n"
-              "<i>Source code available at cpop.tw/code under GPLv3 "
-              "license</i>\n\n" +
+                           "<i>Source code available at cpop.tw/code under GPLv3 "
+                           "license</i>\n\n" +
               CONTACTS_HTML)
 HELP_HTML = (AVAILABILITY_HTML + "\n\n"
-             "<b>Usage</b>:\n"
-             "- send a message that only contains a link of a YouTube video "
-             "or a SoundCloud track.\n"
-             "- Playlists are not supported, but you can select multiple "
-             "audio messages and forward them to create playlist instead.\n"
-             "- Your message will be deleted in private chat after the music "
-             "gets successfully uploaded.\n"
-             "- You can get YouTube links with inline bot @vid and use "
-             "them.\n\n" +
+                                 "<b>Usage</b>:\n"
+                                 "- send a message that only contains a link of a YouTube video "
+                                 "or a SoundCloud track.\n"
+                                 "- Playlists are not supported, but you can select multiple "
+                                 "audio messages and forward them to create playlist instead.\n"
+                                 "- Your message will be deleted in private chat after the music "
+                                 "gets successfully uploaded.\n"
+                                 "- You can get YouTube links with inline bot @vid and use "
+                                 "them.\n\n" +
              CONTACTS_HTML)
```

Comment:
Here it exceed 79 characters length.

Reply:
I think both style is fine (as long as it does not exceed 79 characters), but the string was too long, it waste a lot of space to just indent another level.
If you want it to look better, maybe just this?
```
HELP_HTML = (AVAILABILITY_HTML +
             "\n\n"
             "<b>Usage</b>:\n"
             "- send a message that only contains a link of a YouTube video "
             "or a SoundCloud track.\n"
             "- Playlists are not supported, but you can select multiple "
             "audio messages and forward them to create playlist instead.\n"
             "- Your message will be deleted in private chat after the music "
             "gets successfully uploaded.\n"
             "- You can get YouTube links with inline bot @vid and use "
             "them.\n\n" +
             CONTACTS_HTML)
```

Reply:
oh shit i forgot to check lint

Reply:
actually looks like lint didn't say anything
<img width="772" alt="Screenshot 2020-12-21 at 01 05 10" src="https://user-images.githubusercontent.com/49668442/102720801-0e71ef00-4329-11eb-98c1-1a518beb5f98.png">

Reply:
but i'll fix it tomorrow, i agree it looks too long